cli/src/athenapdf.js

```diff
@@ -34,6 +39,7 @@ athena
     .option("-S, --stdout", "write conversion to stdout")
     .option("-A, --aggressive", "aggressive mode / runs dom-distiller")
     .option("-B, --bypass", "bypasses paywalls on digital publications (experimental feature)")
+    .option("-R, --request-header <key-value pair>", "Request header to send in the format 'key=value'. Both 'key' and 'value' are expected to be urlencoded", collect, [])
```

Comment:
@MrSaints we could call it --http-header as suggested on the PR #72 , which also matches the shorthand `-H` as used by curl.